modules/gapi/include/opencv2/gapi/garg.hpp

```diff
@@ -99,6 +99,14 @@ using GRunArg  = util::variant<
     >;
 using GRunArgs = std::vector<GRunArg>;
 
+// TODO: Think about the addition operator
+inline GRunArgs& operator += (GRunArgs &lhs, const GRunArgs &rhs);
+inline GRunArgs& operator += (GRunArgs &lhs, const GRunArgs &rhs) {
```

Comment:
I believe our code convention is to put `{` on a new line. Please, apply everywhere

Reply:
Done